src/Sulu/Bundle/ContentBundle/Admin/SuluContentContentNavigation.php

```diff
@@ -21,6 +21,12 @@ public function __construct()
 
         $this->setName('Content');
 
+        $this->addNavigationItem($this->getContent());
+
+        $this->addNavigationItem($this->getSeo());
+        $this->addNavigationItem($this->getExcerpt());
+
+        $this->addNavigationItem($this->getSettings());
```

Comment:
Think I just copied it from a different place, will change it. Maybe it was done like this because seo and excerpt are the content extensions.